core/kazoo_globals/src/kz_nodes.erl

```diff
@@ -1071,3 +1072,119 @@ node_encoded() ->
             Encoded;
         {'ok', Encoded} -> Encoded
     end.
+
+-spec whapp_role_count(text(), text()) -> integer().
```

Comment:
Any reason we are not moving these to kapp* functions ?

Reply:
since the others weren't i just followed the pattern